.github/working_agreement.md

```diff
@@ -0,0 +1,171 @@
+    - Ice Box: Items that are unvetted ideas, blocked w/no time estimate for unblocking, ticket that needs fleshing out by reporter, etc.
+    - Backlog (Ready): Stories that have been refined, triaged, and prioritized.
+    - Sprint Backlog: Stories that have been committed for the current sprint.
+    - **In Progress**: A development team member owns the story or bug and begins work.
+    - **In Review**: The owner of the story or bug determines the item meets our Definition of Done and has created a Pull Request. The item will stay in this status through the PR process -- including addressing requested feedback or fixing issues found.
+    - **Closed**: The Pull Request/Task has completed, and the work has been committed to the `main` branch of the project repository.
+
+### Estimating
+
+- We will estimate User Stories with size tagging to help gauge how much we work we commit to within a sprint.
+- We will use T-shirts sizing for our estimation -- with XS <1 day, Small is 1-2 days, Medium is 3-5 days, Large is 5 days, and XL is anything over 5 days (which would need to be broken down).
+
+### Ceremonies
+
+- Our sprints will be one week and run from Thursday - Wednesday with Review, Retrospective, and Planning occurring back-to-back on the last day of sprint.
+- We will use an assigned Scrum Master versus rotating the role among the team.
+
+|  | When | Length | Participants | Purpose |
+|------|------|--------|--------------|---------|
+| **Standup** | Tuesday, Wednesday, Thursday | 10 minutes | Development Team, Scrum Master, Product Owner | Those with committed work answer: What did I do yesterday? What will I do today? Is there anything in my way? |
+| **Parking Lot** | Tuesday, Wednesday, Thursday | 20 minutes | Development Team, Scrum Master, Product Owner| Review an net-new issues for prioritization and discussion |
+| **Sprint Review** | Wednesdays | 30 minutes | Development Team, Scrum Master, Product Owner | Demonstrate the work we did this week. Show and tell time. |
+| **Sprint Retrospective** | Wednesdays | 30 minutes | Development Team, Scrum Master, Product Owner | Reflect as a team on how we're doing -- what's working well for us, and what could we do better? |
+| **Sprint Planning** | Wednesdays | 30 minutes | Scrum Master, Product Owner, Development Team | Commit to work for the next sprint. |
+
+## How we code together
+
+### Branch Strategy
+
+- `main` branch will be used as the production branch
+  - It will be locked, requiring a PR to make commits.
+  - It will be shippable at any time.
+  - Releases will occur through a merge of `dev` into `main`
+- Branches will be used for story or bug-fix work.
+  - Naming convention will include alias with a short description of the story or bug (e.g., `dsturgell-create-working-agreement` or `jofultz-deploy-script-error-on-box`)
+  - Commits should always have a short but descriptive message explaining what is changing.
```

Comment:
Probably worth mention that `dev` is a protected working branch for CI. and `main` should always have ready to release code.